testdata/test-gpg.key

```diff
@@ -0,0 +1,82 @@
+-----BEGIN PGP PRIVATE KEY BLOCK-----
```

Comment:
Just to confirm, this file is intentional, right?

Reply:
Yep this key is used for the `pass` tests

Reply:
https://github.com/openshift-online/ocm-cli/pull/600/files/44aa9785625d0774ac9294783fdf8a73c3a3ada9#diff-ee2c107da56915eab7b161dc907e197d3ef7142c7b0dea278d5f196d9910299eR84